lib/inventory_refresh/persister/mark_and_sweep_mixin.rb

```diff
@@ -0,0 +1,160 @@
+module InventoryRefresh
+  class Persister
+    module MarkAndSweepMixin
+      extend ActiveSupport::Concern
+
+      included do
+        include InventoryRefresh::SaveCollection::Saver::SqlHelper
+      end
+
+      private
+
+      def define_refresh_state_ics
+        refresh_states_inventory_collection = ::InventoryRefresh::InventoryCollection.new(
```

Comment:
This looks like it belongs in the app not in the gem, we're forcing any app that includes this to define models with specific names to handle this.

The calling app should handle the refresh state/parts and just pass in the delete conditions to the gem IMO

Reply:
hm, we'd probably need something in the middle

Reply:
like a Persister gem :-)

Reply:
Haha like some sort of ... topological_inventory-persister?